### ISSUE TITLE
How to use color by value with plots

### ISSUE THREAD

**Issue by chrispikula** [dmnfarrell/pandastable#101]
Hello,
I was wondering, how does one go about using color by value with plots such as h-bar plots?  (Not color by value of the table)
I've tried messing around with things, but, so far, I've had no luck.

**Comment by dmnfarrell** [comment]
Yes color by value is only implemented for scatter plots so far. For bar plots it can be done but I'm not sure exactly what you are after.
How does your explanation differ from simply using grouped plots as in this example:

![test](https://user-images.githubusercontent.com/7859189/45061932-7b9ce000-b09e-11e8-8f86-8dcf60506c2e.png)

**Comment by chrispikula** [comment]
My dataset consists of a part-number, and four columns, high, medium, low and very low counts.  Only the part-number is a string, the rest are numeric. 

I've got several part numbers that I'm grouping by, and for each one of those part numbers, I've got a high, medium, low and very low bar.  Usually.  Sometimes some only have 2 of the four, but that's a different story.

I'd like to have each cluster colored with green representing high, and red representing very low, and ideally in order, but I think for basic workings, I'd be happy with a definable color code.  

Ideally, what I would like is for there to be a way to take two columns of data, the part number and a color-coded numeric value of the counts high/medium/low/very-low.  (Yes, some reorganization would be necessary.)  Then, when you would graph that particular number, the bar would have the color of the cell.

If I've got to do it myself, I'll do it myself, but I was just wondering if there was a trick that I was missing.

Overall, I've been having a blast with this application.  It's great, and I'm going to recommend it to my friends.

**Comment by dmnfarrell** [comment]
If I understand you, you need the bar colors to be determined by the actual cell value using some custom color scale, rather than just as an ordered series. This is definitely not possible at the moment as it's a fairly specific use case. I generally try to implement generic methods that can be of most general use. I may be able to implement a general version of this using the color by value option for bar plots when I get time. If you can code this yourself in python it would likely be faster. If you want to know how to put it in a plugin I can show you. You could then re-use your code inside the app.
Thanks for the feedback ;)

**Comment by chrispikula** [comment]
I'd be up for figuring out how to use it in a plugin.  I'm already doing some simple modifications to the app, and it'd probably be best to put those features in a plugin.  Would it be alright if we continued this via e-mail?  Mine is my username @gmail.com
